content/blog/D0014-openage_mod_api_finale_0.1.md

```diff
@@ -0,0 +1,79 @@
+
+**Proper inventories and item systems**<br/>
+Relics will no longer replace the monk unit when picked up. Instead, the API unifies garrisoning, transporting and inventories with a new container system that give units the ability to store other game entities properly. Items and garrisoned units are allowed to grant new abilities or remove existing ones.
+
+**Enable different abilities depending on the unit's state**<br/>
+Like the trebuchet from AoE2, all game objects can have multiple states which activate other abilities and give different boni or mali. Modders will be able to define as many transformation states as they like. Additionally, construction, damage and harvest progress are able to change the state of the unit. For example, this can be used to make a building's armor weaker when it is not fully constructed.
+
+**Combine effects such as damage, convert or heal**<br/>
+Effects in the API are just as modular as our ability and modifier system. Any unit can apply any effect and is allowed to combine them freely. As an example, modders will be able to create attacks that do damage and also have a chance to convert the unit.
+
+**Techs and civ bonuses affect more diplomatic stances**<br/>
+In AoE, your civ choice grants bonuses to the whole team. But what if your enemies would profit too? In our API, civs are allowed to define modifiers for other players that have a different stance than "friendly". Additionally, techs can influence not only yourself, but also others on the same map. Abilities and effects are also allowed to be limited to specific diplomatic stances which enables more specialized diplomacy such as overlord-vassal relationships.
+
+These are the most prominent features that we support. There are a lot of other helpful features that were either too small or too complex to mention. Be sure to check out our [previous blogpost]({filename}/blog/landing_page.md) if you are interested in more thorough explanations of the underlying mechanics.
+
+## What happens next?
+
+We will stop thinking about new features for a while and integrate the existing API into the engine. The core engine parts that need to be worked on are the *core gamestate* and the *converter*. The gamestate needs to be able to understand the API objects and implement the behavior for abilities and modifiers. Our converter can already read the `.dat` files of the original game and now needs to sort the old data into our nyan API tree.
+
+Roadmap for integration:
+
+* Write the API documentation (this can later be used by modders as a reference sheet)
+* The converter will be changed to create nyan API objects and assemble them as a modpack
+* In parallel, the engine will be extended to load API objects and implement their intended behavior
+* Once everything works, write a modding guide and tutorials
+
+Let's do it!
+
+## Fun facts
+
+As a bonus, here is nice collection of fun and pointless facts about the API and its development.
+
+* **The first API draft:**
```

Comment:
i was confused about this at first, maybe it helps to write something like "our first API draft attempt (September 2008)" :grin:

the *first* api draft sounds like something we've done recently or is the result.